x/stakeibc/keeper/community_pool.go

```diff
@@ -245,12 +245,22 @@ func (k Keeper) BuildFundCommunityPoolMsg(
 			"fund community pool ICA can only be initiated from either the community pool return or withdrawal ICA account")
 	}
 
-	msgs := []proto.Message{&disttypes.MsgFundCommunityPool{
-		Amount:    sdk.NewCoins(token),
-		Depositor: sender,
-	}}
+	// If the community pool treasury address is specified, bank send there
+	if hostZone.CommunityPoolTreasuryAddress != "" {
+		fundMsg = []proto.Message{&banktypes.MsgSend{
+			FromAddress: sender,
+			ToAddress:   hostZone.CommunityPoolTreasuryAddress,
```

Comment:
potential error case: we could mistakenly set `hostZone.CommunityPoolTreasuryAddress` to the address of the actual community pool, which is not whitelisted for bank sends. any way to prevent this case, or should we just be vigilant about setting it properly?

Reply:
I think we should just be vigilant about setting it properly.

If it wasn't whitelisted, I think the ICA would just fail